.github/workflows/latest.yaml

```diff
@@ -7,24 +7,24 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Should be replaced by single workflow file as done in other repos as well (e.g. metal-bmc).

Reply:
done